### PR TITLE
Added the portuguese wiki and update README

### PR DESCRIPTION
**Pull request by Gawiga**
I ran to a problem with the build of the jekyll in my machine, so I added this solution at the readme.
https://github.com/jekyll/jekyll/issues/5267#issuecomment-241379902
I also saw that one commom issues on the site is the improvement of the wiki, so I gave a first try.
I really like this project and I hope to help as much as I can.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Delete Gemfile.lock and run `bundle install`
 bundle exec jekyll serve
 ```
 
+If you have a problem running the command above, add `vendor` at the section exclude in the _config.yml file.
```
Comment (barton2526) [inline README.md:57]: @RoboticMind what are your thoughts on this?

Comment (RoboticMind) [reply README.md:57]: Maybe should just add the config change to _config.yml instead?

Comment (Gawiga) [reply README.md:57]: Done! I added `vendor` at the section exclude in _config.yml file and translated another page to portuguese

Comment (barton2526) [reply README.md:57]: this comment can be removed now, correct?

Comment (Gawiga) [reply README.md]: Yes, now I removed. 😅

**Review by RoboticMind — APPROVED** [review]
uACK

**Pull request merged**